### PR TITLE
responsive navbar

### PR DESCRIPTION
**Pull request by RobinBurri**
The navbar is responsive now, not perfect but decent imo.

There are only 4 links now and, the 42 logo is hidden on small screen.

The "Sign in" page is now '/' and there is no links (only the 42 svg) when you arrive there logged out.

The dir that was called welcome is now called sigin.

RIP ```<Welcome>``` component

### CODE REVIEW

**Review by shaolin-peanut — APPROVED** [review]

**Review by eprei — APPROVED** [review]

File: frontend/src/components/signin/SignInBtn.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { Link } from 'react-router-dom'
+import styles from './SignInBtn.module.css'
+
+const SigneInBtn = () => {
```
Comment (t-h2o) [inline frontend/src/components/signin/SignInBtn.tsx:4]: Is it a typo ?
```suggestion
const SignInBtn = () => {
```

**Review by t-h2o — COMMENTED** [review]
:keyboard:

File: frontend/src/components/signin/SignInBtn.tsx (hunk 1 above)
Comment (RobinBurri) [reply frontend/src/components/signin/SignInBtn.tsx]: yes
f0370ba6ca3cb03ce2542d9d461c70a53f2bb6dc

File: frontend/src/components/signin/SignInBtn.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+const SignInBtn = () => {
+    return (
+        <Link to="/profile" className={styles.btn}>
+            Sign in
+        </Link>
+    )
+}
+
+export default SigneInBtn
```
Comment (t-h2o) [inline frontend/src/components/signin/SignInBtn.tsx:12]: And this one too ?
```suggestion
export default SignInBtn
```

Comment (RobinBurri) [reply frontend/src/components/signin/SignInBtn.tsx]: 0042c2cba0915652b4bafdc546a62eef48dad9a9

**Review by t-h2o — APPROVED** [review]

**Review by fire-poy — APPROVED** [review]

**Pull request merged**